README.markdown

```diff
@@ -7738,6 +7739,31 @@ This method was first introduced in the `v0.5.0rc1` release.
 
 [Back to TOC](#nginx-api-for-lua)
 
+tcpsock:setclientcert
+--------------------
+
+**syntax:** *tcpsock:setclientcert(cert, pkey)*
```

Comment:
```suggestion
**syntax:** *ok, err = tcpsock:setclientcert(cert, pkey)*
```

Reply:
sorry, I forgot it, I will update it soon, thanks.